artifacts/definitions/SUSE/Linux/Events/Timers.yaml

```
@@ -0,0 +1,28 @@
+name: SUSE.Linux.Events.Timers
+description: |
+   Print events when a systemd timer gets added/deleted/executed
+
+type: CLIENT_EVENT
+
+sources:
+  - name: TimerExecs
+    precondition: SELECT OS From info() where OS = 'linux'
+    description: Collect systemd timer executions from journal
+    query: |
+      LET timers = SELECT parse_json_array(data=Stdout) AS list
+        FROM execve(argv=['systemctl', 'list-timers', '--all', '-o', 'json', '--no-pager'])
+      
+      LET timer_exec = SELECT REALTIME_TIMESTAMP, _PID, MESSAGE, SYSLOG_IDENTIFIER, _SYSTEMD_UNIT, _UID, {SELECT activates from timers.list} AS activates
+        FROM watch_journal()
+        WHERE _SYSTEMD_UNIT IN activates
```

this is triggering the `systemctl list-timers ...` every time the journal gets a new message. Can use lazy evaluation so that only runs if the systemd unit name ends with `.service`

```suggestion
        WHERE _SYSTEMD_UNIT =~ ".*service" AND _SYSTEMD_UNIT IN activates
```

---

Done.